keps/NNNN-kep-template/kep.yaml

```diff
@@ -13,9 +13,36 @@ reviewers:
   - "@alice.doe"
 approvers:
   - TBD
-  - "@oscar.doe"
+  - "@oscar.doe" # PRR Approver
 see-also:
   - "/keps/sig-aaa/1234-we-heard-you-like-keps"
   - "/keps/sig-bbb/2345-everyone-gets-a-kep"
 replaces:
   - "/keps/sig-ccc/3456-replaced-kep"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha|beta|stable
```

Comment:
there is no such field in the `Proposals` structure, causing `kepval` / `make verify` to fail:
https://github.com/kubernetes/enhancements/blob/master/pkg/kepval/keps/proposals.go#L49-L76

looks like the template itself is not validated.

Reply:
this has since been fixed, thanks